web/js/containers/projection.js

```diff
@@ -94,6 +95,7 @@ const mapDispatchToProps = (dispatch) => ({
   updateProjection: (id, config) => {
     dispatch(changeProjection(id));
     dispatch(resetProductPickerState(id));
+    dispatch(stop());
```

Comment:
Nit pick... But I would lean towards only firing the dispatcher only if animating.

Reply:
Good call!